api/comment_threads.rb

```diff
@@ -1,21 +1,22 @@
 
 get "#{APIPREFIX}/threads" do # retrieve threads by course
+
   threads = CommentThread.where({"course_id" => params["course_id"]})
```

Comment:
@clintonb: Looks like this params would also use single quotes.  Can you explain where you want to use single vs double quotes?